packages/web-components/src/tabs/tabs.spec.ts

```diff
@@ -0,0 +1,552 @@
+
+    root = page.locator('#root');
+
+    tablist = element.locator('.tablist');
+
+    tabs = element.locator('fluent-tab');
+
+    await page.goto(fixtureURL('components-tabs--tabs-default'));
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('should reset tab indicator offset and scale for horizontal orientation after animation', async () => {
+    await root.evaluate(
+      (node, { template }) => {
+        node.innerHTML = template;
+      },
+      { template },
+    );
+
+    const tab = tabs.nth(2);
+    await tab.click();
+
+    const tabIndicatorValues = await page.evaluate(() => {
+      const tabsElement = document.querySelector('fluent-tabs') as Element;
+      return {
+        offset: getComputedStyle(tabsElement).getPropertyValue('--tabIndicatorOffset').trim(),
+        scale: getComputedStyle(tabsElement).getPropertyValue('--tabIndicatorScale').trim(),
+      };
+    });
+
+    expect(tabIndicatorValues.offset).toBe('0px');
+    expect(tabIndicatorValues.scale).toBe('1');
+  });
+
+  test('should animate the active tab indicator', async () => {
+    await root.evaluate(
+      (node, { template }) => {
+        node.innerHTML = template;
+      },
+      { template },
+    );
+
+    const tab = tabs.nth(2);
+
+    const valueBeforeClick = await page.evaluate(() => {
+      const tabElement = document.querySelector('fluent-tab:nth-child(3)') as Element;
+      return tabElement.getAttribute('data-animate');
+    });
+
+    expect(valueBeforeClick).toBe(null);
+
+    await tab.click();
+
+    const valueAfterClick = await page.evaluate(() => {
+      const tabElement = document.querySelector('fluent-tab:nth-child(3)') as Element;
+      return tabElement.getAttribute('data-animate');
+    });
+
+    expect(valueAfterClick).toBe('true');
+  });
+
+  test('should have reflect disabled attribute on control', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+            <fluent-tabs></fluent-tabs>
+        `;
+    });
+    await expect(element).not.toHaveAttribute('disabled', '');
+
+    await element.evaluate((node: Tabs) => {
+      node.disabled = true;
+    });
+
+    await expect(element).toHaveAttribute('disabled', '');
+  });
+
+  test('should have an internal element with a role of `tablist`', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+              <fluent-tabs></fluent-tabs>
+          `;
+    });
+
+    await expect(tablist).toHaveAttribute('role', 'tablist');
+  });
+
+  test('should set a default orientation value of `horizontal` when `orientation` is not provided', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+                  <fluent-tabs></fluent-tabs>
+              `;
+    });
+
+    await expect(element).toHaveJSProperty('orientation', 'horizontal');
+  });
+
+  test('should set an `id` attribute on the active tab when an `id` is provided', async () => {
```

Comment:
nit: What do you mean by the "active" tab? the test set and tests the `id` on all tabs.